src/js/cookie-message.js

```diff
@@ -111,8 +101,16 @@ class CookieMessage {
 	 */
 	showCookieMessage() {
 		this.cookieMessageElement.classList.add(
-			`${this.options.cookieMessageClass}--active`
+			'o-cookie-message',
```

Comment:
oh, actually, if I remember right I don't think IE supports multiple arguments in classlist

Reply:
hmm, i'll add `Element.prototype.classList` to the `browserFeatures`